spec/restaurant_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'restaurant'
+
+describe Restaurant do 
+  it 'can create an instance of Restaurant' do
+    expect(subject).to be_kind_of(Restaurant)
+  end
+
+  it 'responds to the method .list_menu' do
+    expect(subject).to respond_to(:list_menu)
+  end
+
+  it 'responds to the method .order' do
+    expect(subject).to respond_to(:order).with(2).argument
+  end
+
+  it 'responds to the method .total_basket' do 
+    expect(subject).to respond_to(:total_basket)
+  end
+
+  it 'responds to method .total_price' do
+    expect(subject).to respond_to(:total_price)
+  end
+
+#   describe '#list_menu' do
+#     it 'should output a menu list from the Restaurant' do 
```

Comment:
I don't know if this is an appropriate or even useful test, but there's something where you can check that it sends something to output. expect { subject.list_menu }.to output('optional something').to stdout... or something like that